### PR TITLE
Feature/8 customer rabbit mq

### LINKED ISSUE #8: Set up and persist RabbitMQ

### PR DESCRIPTION
**Pull request by JustMilan**
## Changes
A short description about the overall changes you've made to the project.

## Checklist
- [x] Implement RabbitMQ
- [x] Format all modified files
- [x] Check for missing newlines
- [x] Optimize imports

## When merging, this pull request 
closes #8

### CODE REVIEW

File: customer/pom.xml [hunk 1]
```diff
@@ -17,7 +17,10 @@
 		<java.version>17</java.version>
 		<sonar.organization>xandervedder</sonar.organization>
 		<sonar.host.url>https://sonarcloud.io/</sonar.host.url>
-		<sonar.exclusions>**/CustomerApplication.java</sonar.exclusions>
+		<sonar.exclusions>
+			**/CustomerApplication.java
```
Comment (xandervedder) [inline customer/pom.xml:21]: comma is in the wrong place

File: customer/src/main/java/nl/softwarestrijders/waiter/customer/infrastructure/config/RabbitMqConfig.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+public class RabbitMqConfig {
+	@Value("${spring.rabbitmq.host}")
+	private String host;
+	@Value("${spring.rabbitmq.port}")
+	private int port;
+
+	@Value("${messaging.exchange.waiter}")
+	private String waiterExchangeName;
+	@Value("${messaging.queue.customer}")
+	private String customerMessageQueue;
```
Comment (xandervedder) [inline customer/src/main/java/nl/softwarestrijders/waiter/customer/infrastructure/config/RabbitMqConfig.java:24]: Just a note, in the future we might want to have different queues for different events

**Review by xandervedder — APPROVED** [review]
Basically my code 😆 

looks good 👍

**Review by Rubenvdbrink — APPROVED** [review]
yes

**Pull request merged**